README.rst

```diff
@@ -128,6 +128,10 @@ installation.
 
 Now the system is ready to use. For future sessions, only the step2 is required.
```

Comment:
Well, I can understand that it should be deleted, shouldn't? Only a REAME.md or README.rst